scripts/lib/cluster.sh

```diff
@@ -36,7 +36,7 @@ function up-test-cluster() {
         $TESTER_PATH eks create cluster --enable-prompt=false --path $CLUSTER_CONFIG || (echo "failed!" && exit 1)
     else
         echo -n "Creating cluster $CLUSTER_NAME (this may take ~20 mins. details: tail -f $CLUSTER_MANAGE_LOG_PATH)... "
-        $TESTER_PATH eks create cluster --path $CLUSTER_CONFIG >>$CLUSTER_MANAGE_LOG_PATH 1>&2 ||
+        $TESTER_PATH eks create cluster --enable_prompt=false --path $CLUSTER_CONFIG >>$CLUSTER_MANAGE_LOG_PATH 1>&2 ||
```

Comment:
Here we do want the prompt.

Reply:
Thanks for the review @mogren . Yes it is not needed. I have fixed it in the caller script. Will resubmit the PR with changes.